Automattic-Tracks-iOS/Crash Logging/CrashLogging.swift

```diff
@@ -278,6 +286,104 @@ extension CrashLogging {
     }
 }
 
+// MARK: - Helpers for hybrid SDKs
+extension CrashLogging {
+    /// Returns the options required to initialize Sentry in other platforms.
+    public func getOptionsDict() -> [String: Any] {
+        return [
+            "dsn": self.dataProvider.sentryDSN,
+            "environment": self.dataProvider.buildType,
+            "releaseName": self.dataProvider.releaseName
+        ]
+    }
```

Comment:
On Gutenberg, we need to initialize the Sentry SDK (React native version) so I exposed the required options for this.